render/allocator.c

```diff
@@ -9,6 +18,49 @@ void wlr_allocator_init(struct wlr_allocator *alloc,
 	wl_signal_init(&alloc->events.destroy);
 }
 
+struct wlr_allocator *wlr_allocator_autocreate(struct wlr_backend *backend,
+		struct wlr_renderer *renderer) {
+	wlr_log(WLR_DEBUG, "Picking the appropriate allocator");
```

Comment:
Maybe we can just drop this log line, it doesn't seem like it provides any useful information.